src/bin/hexyl.rs

```diff
@@ -149,6 +151,33 @@ fn run() -> Result<(), AnyhowError> {
                     A negative value is valid and calculates an offset relative to the \
                     end of the file.",
                 ),
+        )
+        .arg(
+            Arg::new("panels")
+                .short('w')
+                .long("panels")
+                .takes_value(true)
+                .value_name("N")
+                .help(
+                    "Sets the number of hex data panels to be displayed. \
+                    `--panels=auto` will display the maximum number of hex data panels \
+                    based on the current terminal width",
+                ),
+        )
+        .arg(
+            Arg::new("terminal_width")
+                .short('t')
```

Comment:
I'm trying to be conservative with short options. There are only a few of them :-). I would suggest that we remove both short options for now (i.e.: same for `-w`).